src/plugins/wb5-AJAX-submit/wb5-AJAX-submit.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @title WET-BOEW wb5-AJAX-submit
+ * @overview This plugin implements AJAX request for form data
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @masterbee, @namjohn920
+ **/
+
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "AJAX-submit",
+	selector = ".provisional.AJAX-submit",
+	initEvent = "wb-init" + selector,
+
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector );
+
+		elm.addEventListener( "submit", function( e ) {
+			e.preventDefault();
+
+			var $form = $( this );
+
+			console.log( "action", $form.attr( "action" ) );
+
+			try {
+				$.ajax( {
+					type: $form.attr( "method" ),
+					url: $form.attr( "action" ),
+					data: $form.serialize() // serializes the form's elements.
+				} );
+			} catch ( error ) {
+				console.error( error );
+			}
+
+			//Insert an appropriate sucess message within quotation mark
+			$form.html( "<h1>Thank you! It is successfully submitted</h1>" );
```

Comment:
Use a plugin config instead. That config could be a jQuery selector that you will happen the text next to the form element.

Reply:
This is also to deal with bilingualism

Reply:
What you should do instead is to have a class named .wb-onpage-thnk on a hidden element underneath the form, and make it show on submit (take the Report a problem on Canada.ca as an example). Also, it would be nice to also have the possibility to show a failure message the same way as the thank yo message for when the AJAX goes wrong. The default could still be the thank you message.